# Put plugins.cfg into /usr/lib/games because it contains an arch-dep path.
Index: opendungeons-0.4.9/CMakeLists.txt
===================================================================
--- opendungeons-0.4.9.orig/CMakeLists.txt	2014-12-22 22:32:10.000000000 +0100
+++ opendungeons-0.4.9/CMakeLists.txt	2015-01-30 23:40:39.279073574 +0100
@@ -32,6 +32,7 @@
     # Set binary and data install locations if we want to use the installer
     set(OD_BIN_PATH ${CMAKE_INSTALL_PREFIX}/games CACHE PATH "Absolute path to the game binary directory")
     set(OD_DATA_PATH ${CMAKE_INSTALL_PREFIX}/share/games/${PROJECT_NAME} CACHE PATH "Absolute path to the game data directory")
+    set(OD_LIB_PATH ${CMAKE_INSTALL_PREFIX}/lib/games/${PROJECT_NAME} CACHE PATH "Absolute path to the game lib directory")
     set(OD_SHARE_PATH ${CMAKE_INSTALL_PREFIX}/share CACHE PATH "Absolute path to the shared data directory (desktop file, icons, etc.)")
 endif()
 
@@ -591,8 +592,9 @@
 ##################################
 
 if(UNIX)
-    set(OD_CONFIGFILES ${CMAKE_BINARY_DIR}/plugins.cfg
-                       ${CMAKE_BINARY_DIR}/resources.cfg)
+    set(OD_CONFIGFILES ${CMAKE_BINARY_DIR}/resources.cfg)
+
+    set(OD_ARCH_CONFIGFILES ${CMAKE_BINARY_DIR}/plugins.cfg)
 
     set(OD_RESOURCES   ${CMAKE_SOURCE_DIR}/config
                        ${CMAKE_SOURCE_DIR}/gui
@@ -613,6 +615,8 @@
             PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
     install(FILES ${OD_CONFIGFILES}
             DESTINATION ${OD_DATA_PATH})
+    install(FILES ${OD_ARCH_CONFIGFILES}
+            DESTINATION ${OD_LIB_PATH})
     install(DIRECTORY ${OD_RESOURCES}
             DESTINATION ${OD_DATA_PATH})
 
